repronim/container/dockercontainer.py

```diff
@@ -80,8 +75,12 @@ def execute_command(self, command, env=None):
             # TODO: might not work - not tested it
             command = ['export %s=%s;' % k for k in command_env.items()] + command
         execute = self._client.exec_create(container=self._container_id, cmd=command)
-        response = [line for line in self._client.exec_start(exec_id=execute['Id'], stream=True)]
-        return response
+
+        for i, line in enumerate(self._client.exec_start(exec_id=execute['Id'], stream=True)):
+            if 'error' in line.lower():
```

Comment:
what is output? if that is just the standard output of the running the command, then we can't rely on sensing 'error' in it (might program might output "there were no error" ;)... or is that some "standardized" output on status of docker running the command?